cli/cmd/start.go

```diff
@@ -265,6 +266,16 @@ func getStartBashCmd(datadir string) (*exec.Cmd, error) {
 	return bashCmd, nil
 }
 
+func getComposeServices() (string, error) {
+	viper := config.Viper()
+
+	services := composeServices["bitcoin"]
+
+	if viper.GetBool("attachLiquid") {
+		services += " " + composeServices["liquid"]
+	}
+}
```

Comment:
missing return statement.
I think the function could return just `string` without `error`